packages/n3h-mock/lib/mem.js

```diff
@@ -26,33 +30,49 @@ const getEntry = exports.getEntry = function getEntry (data) {
   }
 }
 
+// Object for storing & retrieving data
 class Mem {
+
+  // ctor
   constructor () {
+    // main datastore
     this._data = {}
+    // array of indexers: store with an indexing function
     this._indexers = []
   }
 
+  // add an indexer
+  // input: indexing function
+  // return the indexer's store
```

Comment:
generally follow the [JSDoc](http://usejsdoc.org/) syntax for inline javascript api comments

so, would be

```javascript
/**
 * add an indexer
 * @param {function} fn - indexing function
 * @return {object} - the indexer's store
 */
```

Reply:
done